http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1,16 +1,13 @@
-swagger: '2.0'
```

Comment:
this conversion was done via a tool that auto converts swagger 2 to openAPI 3

Reply:
Can you run it though an openapi validator tool to be safe?

Also should we confirm this doesn't affect AoU? Might be good to run a test of their code-gen if possible.

Reply:
I opened it in vscode where I have swagger/openapi 3 linter installed, and it looks fine. I slacked @calbach  and @jmthibault79 a few days ago and they confirmed there's no concerns.

@calbach @jmthibault79 can you guys run some codegen that you guys use against the new spec? or let me know which codegen you guys are using, and I can validate

Reply:
No errors on http://editor.swagger.io 👍